src/main/java/core/basesyntax/constructors/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.constructors;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball(new
+               ColorSupplier().getRandomColor(), new
+               Random().nextInt(100));

```

Comment:
The same problem as described in the previous comment, ColorSuplier instance should also be an instance-level variable. It's better not to add a line break after 'new' keyword. And avoid using magic numbers :)

Reply:
Hi mate! thanks for the proper advice's. Will try to fix it all!